spec/spec_helper.rb

```diff
@@ -81,6 +85,12 @@
       Services.register(:logger) { old_logger }
     end
   end
+
+  config.around(:each, type: "sidekiq_fake") do |example|
+    Sidekiq::Testing.fake! do
```

Comment:
Tests tagged with `type: "sidekiq_fake"` that use `Sidekiq::Testing.fake!` here allow us to test that the correct job gets queued with the correct arguments (i.e. for testing phctl itself) without running the job.